src/interpret.rs

```diff
@@ -305,6 +305,76 @@ impl ExecEnv {
                 // here at some point. ~~Alex
                 exit(random());
             }
+            StmtKind::Rickroll => {
+                // possibly move the entire rickroll string into a constant? ~~HTGAzureX1212.
```

Comment:
I would recommend moving to separate file and include using macro `include_str!`